lib/ci/ci_type_parser.js

```diff
@@ -81,6 +82,12 @@ const CI_TYPES = new Map([
     pattern: /job\/node-test-pull-request-lite\/(\d+)/,
     type: JOB_CI | LITE_CI
   }],
+  [LITE_PR_PIPELINE, {
+    name: 'Lite PR Pipeline',
+    jobName: 'node-test-pull-request-lite-pipeline',
+    pattern: /job\/node-test-pull-request-lite-pipeline\/(\d+)\/pipeline/,
+    type: JOB_CI | LITE_CI
```

Comment:
Can you remove `JOB_CI` here? It's not a job but a pipeline.

Reply:
Ah..., done!